src/components/ProjectSearch/TextSearch.js

```diff
@@ -49,6 +82,19 @@ export default class TextSearch extends Component {
     }
   }
 
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.results !== this.props.results) {
+      if (this.focusedItem === null) {
+        return;
+      }
+      const { file, match } = this.focusedItem;
+
+      this.focusedItem[
+        file && !match ? "file" : "match"
```

Comment:
in the case of `!file && !match` we would have the string `match` -- is this what is expected? We might want to move this ternary out into a variable